src/hooks/useDrafts/utils.ts

```diff
@@ -22,7 +22,7 @@ export const addDraft = async (draftProps: Omit<Draft, 'updatedAt' | 'id'>) => {
   return newDraft
 }
 
-export const updateDraft = async (draftId: string, draftProps: RawDraft) => {
+export const updateDraft = async (draftId: string, draftProps: Omit<RawDraft, 'channelId'>) => {
```

Comment:
Why does draft not have the channel information now? Is this captured in some other way? We wouldn't want the drafts to be global across channels right?